src/hotspot/share/code/nmethod.hpp

```diff
@@ -280,7 +277,9 @@ class nmethod : public CodeBlob {
           _has_method_handle_invokes:1,// Has this method MethodHandle invokes?
           _has_wide_vectors:1,         // Preserve wide vectors at safepoints
           _has_monitors:1,             // Fastpath monitor detection for continuations
-          _has_flushed_dependencies:1; // Used for maintenance of dependencies (under CodeCache_lock)
+          _has_flushed_dependencies:1, // Used for maintenance of dependencies (under CodeCache_lock)
+          _is_unlinked:1,              // mark during class unloading
+          _load_reported:1;            // used by jvmti to track if an event has been posted for this nmethod
```

Comment:
It seems like the type could be changed from uint8_t to bool.

Reply:
Is there difference in generated code when you use bool instead of uint8_t?
I used uint8_t to easy change to uint16_t in a future if needed.

Reply:
I don't think uint8_t vs uint16_t matters, only if it is signed, unsigned, or bool.  So if we have more than 8 individual :1 fields, it will expand to a 2nd byte.